src/redisearch_rs/build_utils/src/lib.rs

```diff
@@ -38,7 +38,7 @@ fn rerun_if_changes(dir: &Path, extensions: &[&str]) -> std::io::Result<()> {
         } else if let Some(extension) = path.extension().and_then(|ext| ext.to_str())
             && extensions.contains(&extension)
         {
-            println!("cargo:rerun-if-changed={}", path.display());
+            println!("cargo::rerun-if-changed={}", path.display());
```

Comment:
Looks like this syntax has been introduced in `1.77`. Should we define it as MSRV? We do not have `rust-version` yet in the workspace `Cargo.toml`.

Reply:
We're not a library, so the effective MSRV is what we use in our `rust-toolchain.toml` file.